build/yamls/flow-visibility/grafana/provisioning/datasources/create_table.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
```

Comment:
Where this file is used? I do not see that.

Reply:
It's used at here to create the data-schema configmap: https://github.com/antrea-io/antrea/blob/4f2150eaffc7eaa28ce49de2607fe7511504a4ed/build/yamls/flow-visibility/start-flow-visibility.sh#L25